src/chapters/thymeleaf-views/using-templates.rst

```diff
@@ -0,0 +1,182 @@
+Using a Template
+=================
+
+The video on this page provides you some more live-coding practice with
+Thymeleaf templates. Return to your ``hello-spring`` project and code along as
+you watch the clip.
+
+.. admonition:: Warning
+
+   This video and the others in the chapter walk you through building a small
+   web-based project. Do NOT skip this practice, because the end of chapter
+   exercises pick up where the tutorials end.
+
+Try It!
+--------
+
+.. todo:: ``hello-spring`` video #2 goes here...
+
+Now that your project is growing, let's review one skill and two
+best-practices.
```

Comment:
I went back and forth about whether to put the videos first or last on the pages.  I kept debating if it is better for the students to get the practice first and then review specific details, or if they should get the text first and then do the practice.  Deductive vs. inductive scenario.  (This might be classified as a "philosophical discussion").

On the one hand, reading the text first makes the videos easier to follow, but having the info up front might discourage watching the clips.  I ran in circles trying to decide which way to go.

Can we touch base on this?

Reply:
OK.  I moved the video to the bottom of the page, and I updated the text to accommodate the change.

Let me know if the start-of-page intro needs work.